db/migrate/20180626121716_add_uploaded_to_asset_manager_at_time_to_attachment_data.rb

```diff
@@ -0,0 +1,5 @@
+class AddUploadedToAssetManagerAtTimeToAttachmentData < ActiveRecord::Migration[5.0]
+  def change
+    add_column :attachment_data, :uploaded_to_asset_manager_at, :datetime, default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
Should this definitely have a default? This would mean that creating new instances of this object without specifying this would have this field set. It might be better to leave the default as null but include a second part of this migration which sets the time to now for all existing attachment datas.

Reply:
Yes, I think that's the better thing to do.  In fact, setting a default here is definitely wrong - but we do want all existing `AttachmentData`s to have a default.